### ISSUE TITLE
Undocumented Zend\Form\FieldsetInterface return type for the Zend\Form\Fieldset::get() method

### ISSUE THREAD

**Issue by nuxwin** [zendframework/zend-form#205]
Undocumented return type for the Zend\Form\Fieldset::get() method.

The `Zend\Form\Fieldset::get()`  method can return either a `Zend\Form\ElementInterface` or `Zend\Form\FieldsetInterface`. Currently, the `Zend\Form\FieldsetInterface` return type isn't documented leading to IDE (PhpStorm) inspection error:

![warning](https://user-images.githubusercontent.com/125594/41200320-bd9883aa-6ca2-11e8-8672-ef81112bdfac.png)

Basically

![unexpected](https://user-images.githubusercontent.com/125594/41200330-f88f9c0a-6ca2-11e8-8649-43a4fe9c7499.png)

Should look like

![expected](https://user-images.githubusercontent.com/125594/41200334-1320aabe-6ca3-11e8-85b5-81126a4f723d.png)

**Comment by unckleg** [comment]
Help wanted ? :dango: 
@nuxwin would you make PR for this ?

**Comment by nuxwin** [comment]
@unckleg 

I could yes.
